### PR TITLE
IC-1156: Add NPS Region to intervention

### PR DESCRIPTION
**Pull request by Gweaton**
## What does this pull request do?

Adds Region line to Intervention details summary in the Find journey. This isn't filterable, just for the PP's information.

## What is the intent behind these changes?

To display region to PPs looking for an intervention.

## Screenshot

image

### CODE REVIEW

File: server/routes/findInterventions/searchResultsPresenter.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default class SearchResultsPresenter {
   constructor(
     private readonly interventions: Intervention[],
     private readonly filter: InterventionsFilter,
-    private readonly pccRegions: PCCRegion[]
```
Comment (tomsmyers) [inline server/routes/findInterventions/searchResultsPresenter.ts:10]: should these classes now have both region and location?

Comment (Gweaton) [reply server/routes/findInterventions/searchResultsPresenter.ts:10]: Rather than PCC Region and NPS region?

**Review by tomsmyers — APPROVED** [review]
nice

**Comment by Gweaton** [comment]
@tomsmyers @lawrence-forooghian Have updated this to use individual `NPSRegion` and `PCCRegion` rather than shared `Region` to keep things separate and avoid confusion.

**Pull request merged**